addons/binding/org.openhab.binding.tplinksmarthome/ESH-INF/thing/LB100.xml

```diff
@@ -6,9 +6,10 @@
 	<thing-type id="lb100">
 		<label>LB100</label>
 		<description>TP-Link LB100 Smart Wi-Fi LED Bulb with Dimmable Light</description>
+		<category>Lightbulb</category>
 
 		<channels>
-			<channel id="brightness" typeId="brightness" />
+	<channel id="brightness" typeId="brightness" />
```

Comment:
Please keep the tabs intact

Reply:
Ahhh. I keep messing up the tabs in the xml files... There were some other as well...